src/components/FlatList/MVCPFlatList.js

```diff
@@ -137,6 +138,9 @@ const MVCPFlatList = React.forwardRef(({maintainVisibleContentPosition, horizont
     }, [adjustForMaintainVisibleContentPosition, getContentView, getScrollOffset, scrollToOffset]);
 
     React.useEffect(() => {
+        if (!isListRenderedRef.current) {
```

Comment:
@dukenv0307 I think we should add `isListRenderedRef.current` is dependencyList to make sure this code gets execute at least once.

Reply:
Thanks for the update @dukenv0307 

> So the difference is only in In initial render of the component (before the list renders, causing this bug). This is not needed and won't cause any issue because the same logic will be run in onRef [here](https://github.com/Expensify/App/blob/main/src/components/FlatList/MVCPFlatList.js#L156-L157) after the list renders.

In that case, we are good to go.